setup.py

```diff
@@ -88,6 +88,6 @@
     test_suite='tests',
     tests_require=test_requirements,
     url='https://github.com/ska-sa/codex-africanus',
-    version='0.2.9',
+    version='0.2.8',
```

Comment:
Apologies this was a mistake on my part the correct change is 

```suggestion
    version='0.2.9',
```